.github/scripts/verifyActions.sh

```diff
@@ -7,11 +7,6 @@
 declare -r GREEN='\033[0;32m'
 declare -r RED='\033[0;31m'
 declare -r NC='\033[0m'
-declare diff_highlight
```

Comment:
This is unused, so I'm removing it. There is a separate external issue to improve diff highlighting [here](https://github.com/Expensify/Expensify/issues/155392)